elfo-core/src/messages.rs

```diff
@@ -52,6 +52,20 @@ impl<R: Display> From<R> for ConfigRejected {
     }
 }
 
+#[message(ret = Result<(), EntrypointFailed>)]
+#[derive(Constructor)]
+#[non_exhaustive]
+pub struct StartEntrypoint {
```

Comment:
I think we have to stick to the same style in elfo. E.g. in the configurer's protocol: `ReloadConfigs` and `ReloadConfigsRejected`.